src/secure-value.ts

```diff
@@ -229,25 +347,30 @@ export const serializedSecureProxyFactory = (function secureProxyFactory(rawEnv:
                 let secError;
                 const { message, constructor } = e;
                 try {
-                    secError = construct(rawEnv.getSecureRef(constructor), [message]);
+                    // the error constructor must be a raw error since it occur when calling
```

Comment:
This pattern repeats, would be nice if possible to have our own transpilation/codemod to use `rawEnv.getSecureRef` at the source level.